Uplift/Views/Classes/NextSessionCell.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  NextSessionCell.swift
+//  Uplift
+//
+//  Created by Caitlyn Jin on 4/28/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+/// The cell representing a next session of a ClassInstance used in the classes page.
+struct NextSessionCell: View {
+
+    // MARK: - Properties
+
+    let `class`: ClassInstance
```

Comment:
YEaaaaah, definitely try renaming `Class` and `ClassInstance`, escaping the keyword with ` may get annoying later on lol